test-clients/src/main/java/com/hedera/services/bdd/suites/file/ValidateNewAddressBook.java

```diff
@@ -0,0 +1,77 @@
+package com.hedera.services.bdd.suites.file;
```

Comment:
This complete file can be removed as this was added before testing with yahcli, and the content is similar to `FetchSystemFiles.java`